build.sh

```diff
@@ -161,19 +166,19 @@ if (( ${NUMARGS} == 0 )) || hasArg libcudf; then
 fi
 
 # Build and install the cudf Python package
-if (( ${NUMARGS} == 0 )) || hasArg cudf; then
+if buildAll || hasArg cudf; then
 
     cd ${REPODIR}/python/cudf
     if [[ ${INSTALL_TARGET} != "" ]]; then
-    python setup.py build_ext --inplace
-    python setup.py install --single-version-externally-managed --record=record.txt
+        python setup.py build_ext --inplace
+        python setup.py install --single-version-externally-managed --record=record.txt
     else
-    python setup.py build_ext --inplace --library-dir=${LIBCUDF_BUILD_DIR}
+        python setup.py build_ext --inplace --library-dir=${LIBCUDF_BUILD_DIR}
     fi
 fi
 
 # Build and install the dask_cudf Python package
-if (( ${NUMARGS} == 0 )) || hasArg dask_cudf; then
+if buildAll || hasArg dask_cudf; then
 
     cd ${REPODIR}/python/dask_cudf
     python setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
@dillon-cullinan  Why `dask_cudf` does not have `if [[ ${INSTALL_TARGET} != "" ]]; ` check?

Reply:
Not sure, seems like an oversight. @rlratzel was this intended or just a mishap?